node/server/grpc_server.py

```diff
@@ -380,13 +382,13 @@ def __init__(
         self.shutdown_event = asyncio.Event()
 
         options = [
-            ("grpc.max_send_message_length", 100 * 1024 * 1024),  # 100MB
-            ("grpc.max_receive_message_length", 100 * 1024 * 1024),  # 100MB
-            ("grpc.keepalive_time_ms", 60 * 1000),  # 60 seconds
-            ("grpc.keepalive_timeout_ms", 60 * 1000),  # 60 seconds
+            ("grpc.max_send_message_length", 100 * 1024 * 1024),
+            ("grpc.max_receive_message_length", 100 * 1024 * 1024),
+            ("grpc.keepalive_time_ms", 30 * 60 * 1000),       # 30 minutes
+            ("grpc.keepalive_timeout_ms", 30 * 60 * 1000),     # 30 minutes
```

Comment:
Be aware extended timeout limit likely entails loads of bookkeeping with a large numner of connections.

Reply:
any suggestions for what we should use, @enricorotundo?